src/apis/study-room/participate-study.ts

```diff
@@ -0,0 +1,15 @@
+import { http } from '../api';
+
+interface DataType {
+  userId: number;
+}
```

Comment:
이름은 조금 더 명확하게 지어주는게 좋을 것 같네요!
```suggestion
interface ParticipateStudyResponse {
  userId: number;
}
```

Reply:
파일 하나당 하나의 요청만 다루고 있어서 이름을 명확하게 짓지 않아도 의도 파악이 어렵지 않을 것 같아 제가 작성한 api파일에선 모두 `DataType`을 사용하고 있습니다 🙂 그래도 명확한게 더 좋을까요??

Reply:
저는 그래도 명확한게 더 좋다고 생각해요. 
코드를 처음 읽는 사람에게는 코드를 읽는 비용이 발생할 것 같아요.